decidim-elections/app/commands/decidim/votings/check_census.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # A command to check if census data is given
+    class CheckCensus < Rectify::Command
+      def initialize(form)
+        @form = form
+      end
+
+      # Executes the command. Broadcast this events:
+      # - :ok when census is found
+      # - :invalid when the form is not valid
+      # - :not_found when census is not found
+      #
+      # Returns nothing.
+      def call
+        return broadcast(:invalid) unless form.valid?
+
+        check_census!
```

Comment:
Just a personal preference but I'd rather use `check_sensus` without the `!` when the method doesn't have a counterpart with the `!` to denote a special usage 😄 (i.e. you have `create` and `create!` and the `!` raises an exception).

Reply:
Makes sense. It's more a habit, I guess. Thanks for pointing that out! I'll remove it. 😊